pylintfileheadertest/file_header_checker_test.py

```diff
@@ -71,3 +62,22 @@ def test_do_not_ignore_empty_files(self):
                 line=1,
                 args='# Valid\n# Header')):
             self.checker.process_module(node_mock)
+
+
+class TestFileHeaderCheckerNoConfig(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = FileHeaderChecker
+    CONFIG = {}
+
+    def test_no_message_added(self):
+        """When the `file-header` option is not set, no message should be added."""
+
+        self.checker.config.file_header = None
+        node_mock = MagicMock()
+        node_mock.stream.return_value.__enter__.return_value.read.return_value.decode.return_value = '# Invalid\n# Header'
+        with self.assertNoMessages():
+            self.checker.process_module(node_mock)
+
+
+class TestFileHeaderCheckerPath(TestFileHeaderChecker):
```

Comment:
do we need a test method here?

Reply:
nope, all tests from the `TestFileHeaderChecker` are used here, but with the different `CONFIG`

Reply:
but let me add an negative case and case for both config options